### DISCUSSION TITLE
It's not reasonable that log in is required even for accessing a single simple page.

### DISCUSSION THREAD

**Discussion by recursively** [illacloud/illa-builder#1232] [General]
It's a little inconvenient and unreasonable for any user who just wants to access a simple page, users are always required to log in first to access any resources. Is it possible that expose some pages that don't need to log in? In summary, it would be great if I can decide which pages can be accessed directly and which pages are banned without login. In some cases, it's pretty important and necessary!

**Comment by wenzhuo2022** [comment]
Based on the currently supported functions, we recommend that you build a new app and set it as a public app for users outside your team to access. In the future, we will support to configure the access rights of different pages by the developer, but this will take a long time.

**Reply by recursively** [reply]
> Based on the currently supported functions, we recommend that you build a new app and set it as a public app for users outside your team to access. In the future, we will support to configure the access rights of different pages by the developer, but this will take a long time.

Thanks, making it public is a temporary workaround anyway.
